src/components/SearchResults.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import moment from "moment";
+
+export class TableRow extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false,
+      style: {}
+    };
+  }
+
+  onClicked = () => {
+    this.setState(() => {
+      if (this.state.isClicked === false)
+        return {
+          isClicked: !this.state.isClicked,
+          style: { backgroundColor: "skyblue" }
+        };
+      else {
+        if (this.state.isClicked === true)
+          return {
+            isClicked: !this.state.isClicked,
+            style: { backgroundColor: "" }
+          };
+      }
+    });
+  };
+
+  render() {
+    return (
+      <tr
+        className={this.props.item.id}
+        key={this.props.item.id}
+        onClick={this.onClicked}
+        style={this.state.style}
+      >
+        <td>{this.props.item.id}</td>
+        <td>{this.props.item.title}</td>
+        <td>{this.props.item.firstName}</td>
+        <td>{this.props.item.surname}</td>
+        <td>{this.props.item.email}</td>
+        <td>{this.props.item.roomId}</td>
+        <td>{this.props.item.checkInDate}</td>
+        <td>{this.props.item.checkOutDate}</td>
+        <td>
+          {moment(this.props.item.checkOutDate).diff(
+            moment(this.props.item.checkInDate),
+            "days"
+          )}
+        </td>
+      </tr>
+    );
+  }
+}
+export class SearcResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      style: { backgroundColor: "" },
+      sortType: "",
+      sortAscending: true
+    };
+  }
+  onClicking = () => {
+    this.setState({ style: { backgroundColor: "skyblue" } });
+  };
+  compareFunction = (a, b) => {
+    if (a[this.state.sortType] > b[this.state.sortType]) return 1;
+    if (a[this.state.sortType] < b[this.state.sortType]) return -1;
+    return 0;
+  };
+  handleClickToSort = event => {
+    let text = event.target.textContent;
+    text = text.charAt(0).toLowerCase() + text.slice(1);
+    this.setState({
+      sortType: text,
+      changeArowDirection: !this.state.changeArowDirection,
```

Comment:
Do you need this state? I cannot find where it is used.

Reply:
no it was an unsuccessful  try to add  up and down arrow   changing with the sorting